examples/faasd.yaml

```diff
@@ -0,0 +1,69 @@
+# Deploy faasd (which installs a bundled containerd).
+#
+# It can be accessed from the host by authenticating with faas-cli;
+# the ports are already forwarded automatically by lima:
+#
+# $ export OPENFAAS_URL=http://localhost:8080
+# $ limactl shell faasd sudo cat /var/lib/faasd/secrets/basic-auth-password | faas-cli login -u admin --password-stdin
+# $ faas-cli version
+#                 _____           ____
+#  / _ \ _ __   ___ _ __ |  ___|_ _  __ _/ ___|
+# | | | | '_ \ / _ \ '_ \| |_ / _` |/ _` \___ \
+# | |_| | |_) |  __/ | | |  _| (_| | (_| |___) |
+#  \___/| .__/ \___|_| |_|_|  \__,_|\__,_|____/
+#       |_|
+#
+# CLI:
+#  commit:  72816d486cf76c3089b915dfb0b66b85cf096634
+#  version: 0.13.13
+#
+# Gateway
+#  uri:     http://localhost:8080
+#  version: 0.21.0
+#  sha:     539f0a2c946ee6096ef2b6ab26f81295eed49d54
+#
+#
+# Provider
+#  name:          faasd
+#  orchestration: containerd
+#  version:       0.14.3
+#  sha:           ea62c1b12dd1ae1de794e6dd260351cfbd1a6759
+# Your faas-cli version (0.13.13) may be out of date. Version: 0.13.15 is now available on GitHub.
```

Comment:
I think you want to remove the banner and the version, so that the yaml stays relevant for longer ?

Reply:
yes, you're right, I've changed it with another example command to make it more future proof and less verbose